infra/modules/dns-config/main.tf

```diff
@@ -0,0 +1,72 @@
+# referencing the record that already exists in the console
+data "aws_route53_zone" "project_url" {
+  name = "wic-services.org"
+}
+
+data "aws_lb" "participant_alb" {
+  name = var.participant_service_name
+}
+data "aws_lb" "staff_alb" {
+  name = var.staff_service_name
+}
+data "aws_lb" "analytics_alb" {
+  name = var.analytics_service_name
+}
```

Comment:
This module could get abstracted one layer up so that the dns-config module is only:

```
# referencing the record that already exists in the console
data "aws_route53_zone" "project_url" {
  name = "wic-services.org"
}

resource "aws_route53_record" "alias_record" {
  name    = var.url
  type    = "A"
  zone_id = data.aws_route53_zone.project_url.id
  alias {
    name                   = var.alb_dns_name
    zone_id                = var.alb_zone_id
    evaluate_target_health = true
  }
}
```

and then we add the following outputs to service like:

```
output "alb_dns_name" {
  value = aws_lb.alb.dns_name
}

output "alb_zone_id" {
  value = aws_lb.alb.zone_id
}
```

and then in the env-template we do something like:

```
module "participant_dns_record {
  url = var.participant_url
  alb_dns_name = module.participant.alb_dns_name
  alb_zone_id = module.participant.alb_zone_id
}

module "staff_dns_record { 
  ...
}

module "analytics_dns_record {
  ...
}
```

This would result in DRYer code and more reusable modules, especially since the dns records don't actually vary between participant/staff/analytics other than those three variables.